content/tutorial/godot/2D/normal-map-hints/index.md

```diff
@@ -0,0 +1,224 @@
+
++++
+title = "Normal Map Hints"
+description = "Lear how to use Normal Maps with Godot for 2D games."
+author = "Pablo"
```

Comment:
Should be a lowercase id. You need to create an author data file in data/authors too, duplicate one that's there.